typedoc.js

```diff
@@ -3,9 +3,10 @@ module.exports = {
   exclude: [],
   excludeExternals: true,
   excludeNotExported: true,
+  excludeProtected: true,
   excludePrivate: true,
   hideGenerator: true,
-  includeDeclarations: true,
+  includeDeclarations: false,
```

Comment:
Why remove those? If the npm package uses the compiled JS, then these will add TS defs to those compiled JS.

Reply:
Trying to get this crap under control - https://polkadot.js.org/api/api/classes/_promise_index_.apipromise.html - maybe not the best approach